src/bootstrap/Cargo.toml

```diff
@@ -45,15 +45,15 @@ serde = { version = "1.0.8", features = ["derive"] }
 serde_json = "1.0.2"
 toml = "0.5"
 lazy_static = "1.3.0"
-time = "0.1"
+time = { version = "0.3", default-features = false, features = ["std", "formatting"] }
 ignore = "0.4.10"
 opener = "0.5"
 merge = "0.1.0"
 once_cell = "1.7.2"
 
 [target.'cfg(windows)'.dependencies.winapi]
 version = "0.3"
-features = ["fileapi", "ioapiset", "jobapi2", "handleapi", "winioctl", "psapi", "impl-default"]
+features = ["fileapi", "ioapiset", "jobapi2", "handleapi", "winioctl", "psapi", "impl-default", "timezoneapi"]
```

Comment:
This started saying about not found `winapi::um::timezoneapi` until added this feature. Looks like previously it was imported via `time` crate?